config/sys.config

```diff
@@ -36,6 +36,7 @@
   ]},
  {blockchain,
   [
+   {jsonrpc_port, 4467},
```

Comment:
Should this not be off by default?

Reply:
should this not be _the_ way to interact with miner?

Reply:
I'm with @madninja

Reply:
I think we should turn it _on_ by default but only bind to localhost. That seems like the least surprising and "secure" default to me. Other opinions are welcome, though :)